equinox/nn/_normalisation.py

```diff
@@ -94,6 +94,10 @@ def __init__(
             )
         self.use_weight = use_weight
         self.use_bias = use_bias
+
+        with jax.numpy_dtype_promotion("standard"):
+            dtype = jnp.result_type(dtype, jnp.float32)
```

Comment:
This one seems wrong to me. It's totally valid for a user to want to set the weight and bias to a lower dtype, if they really want to.

Reply:
Makes sense. Changed it.